README.md

```diff
@@ -47,17 +49,17 @@ See test/spec/protractor-sync_test.ts for more examples.
 
 # How to contribute
  
-Thanks you for your interest in protractor-sync.  In lieu of a formal styleguide, take care to maintain the existing coding style. Please add tests for any new or changed functionality. 
+Thanks you for your interest in Protractor-sync.  In lieu of a formal style guide, take care to maintain the existing coding style. Please add tests for any new or changed functionality.
```

Comment:
Ha. I didn't know either! As you can see I changed it to be capitalized. Perhaps you or someone else will change it back later. :)